packages/vite/src/node/__tests__/plugins/import.spec.ts

```diff
@@ -75,10 +75,7 @@ describe('transformCjsImport', () => {
       ),
     ).toBe(
       'import __vite__cjsImport0_react from "./node_modules/.vite/deps/react.js"; ' +
-        'const react = ((m) => m?.__esModule ? m : {\n' +
-        '\t...typeof m === "object" && !Array.isArray(m) || typeof m === "function" ? m : {},\n' +
-        '\tdefault: m\n' +
-        '})(__vite__cjsImport0_react)',
+        'const react = ((m) => m?.__esModule ? m : { ...typeof m === "object" && !Array.isArray(m) || typeof m === "function" ? m : {}, default: m })(__vite__cjsImport0_react)',
```

Comment:
It is wired for it coudle be success at rolldown-vite branch.

Reply:
It's probably caused by the `"vitest>vite"` overrides. Vitest uses Vite to process the files and that changes the `Function::toString` result.